BF-it.py

```diff
@@ -32,7 +32,10 @@ def compile_file(input_file, output_file, run):
     with open(input_file, "rt", encoding="unicode_escape") as f:
         code = f.read()
 
-    brainfuck_code = Compiler.compile(code)
+    with open(input_file, "rt") as f:
```

Comment:
redundant to open and read twice and pass both all the way through
can read once and encode/decode when necessary